### DISCUSSION TITLE
Third-party cookie

### DISCUSSION THREAD

**Discussion by rrmini1** [yandex-maps-unofficial/vue-yandex-maps#311] [Q&A (2.x)]
Здравствуйте. С использованием vue-yandex-maps появилось предупреждение: 
Third-party cookie will be blocked. Learn more in the Issues tab.
Как можно решить эту проблему ? Браузер Chrome

**Comment by daniluk4000** [comment]
Добрый день. Это куки с домена Яндекса. Предупреждение можно игнорировать

![image](https://github.com/yandex-maps-unofficial/vue-yandex-maps/assets/10450717/e60d8921-f6ea-432a-b853-7ec84dd070bf)
